README.md

```diff
@@ -17,9 +27,33 @@ Requirements:
 
 - Mamba: [Installation guide](https://mamba.readthedocs.io/en/latest/installation.html) (suggested Mambaforge).
 
-Install the orchestrator virtual environment.
```

Comment:
Use cases should be self-contained, thus use case-specific operations should be automatized and under the specific use case folder